packages/protocol/contracts/L1/TaikoData.sol

```diff
@@ -71,6 +76,8 @@ library TaikoData {
         bytes32 l1Hash;
         bytes32 mixHash;
         bytes32 txListHash;
+        uint32 txStartIdx;
```

Comment:
Brecht, the start/end index can either be the byte-index or the transaction index. Which one is easier for circuits?